AdminWebsite/AdminWebsite/ClientApp/src/app/bookings-list/booking-details/booking-details.component.ts

```diff
@@ -141,7 +141,12 @@ export class BookingDetailsComponent implements OnInit, OnDestroy {
         this.hearing.Endpoints = this.bookingDetailsService.mapBookingEndpoints(hearingResponse);
         this.videoHearingService
             .getAllocatedCsoForHearing(hearingResponse.id)
-            .subscribe(response => (this.hearing.AllocatedTo = response?.cso?.username ?? 'Unallocated'));
+            .subscribe(
```

Comment:
do we put the logic in the client here?

Reply:
yes, for now this is the accepted solution until we can version APIs to avoid breaking S&L

Reply:
should it be Not Allocated as in booking api?